base/broadcast.jl

```diff
@@ -10,6 +10,11 @@ export broadcast_getindex, broadcast_setindex!
 
 ## Broadcasting utilities ##
 
+# fallback routines for broadcasting with no arguments or with scalars
+# to just produce a scalar result:
+broadcast(f) = f()
+broadcast(f, x::Number...) = f(x...)
```

Comment:
thanks, I was attempting some things and not able to get it to work - https://github.com/JuliaLang/julia/commit/d8f4b6073342b2688cc3dff5be94ecc03138c15e wasn't where I thought bisect would point me